libvast/vast/system/source.hpp

```diff
@@ -164,11 +164,11 @@ struct source_state {
           };
           // First, merge and de-duplicate the local schema with types from the
           // type-registry.
-          types.value.insert(std::make_move_iterator(st.local_schema.begin()),
-                             std::make_move_iterator(st.local_schema.end()));
+          types.insert(std::make_move_iterator(st.local_schema.begin()),
+                       std::make_move_iterator(st.local_schema.end()));
```

Comment:
This and the `clear()` in the next line have the effect that a modified `*.schema` file that is passed to the `--schema-file=` option will not be considered. I know this is not a result of this change, but I just stumbled over this while testing this change.